HeliPort/NetworkManager.swift

```diff
@@ -59,7 +59,6 @@ final class NetworkManager {
             if let savedNetworkAuth = CredentialsManager.instance.get(networkInfo) {
                 networkInfo.auth = savedNetworkAuth
                 Log.debug("Connecting to network \(networkInfo.ssid) with saved password")
-                CredentialsManager.instance.setAutoJoin(networkInfo.ssid, true)
```

Comment:
I’m not certain if this is what Apple’s implementation used to do back in the days (feels unlikely and doesn't match with the current implementation). This is the root cause of #243.